src/database/migrations/20220920084207-create-redirects.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("redirections", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      launch_id: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
+      type: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      source: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      target: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      created_at: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updated_at: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
```

Comment:
Minor nit: I think there might be some value in adding a `defaultValue: Sequelize.fn("NOW")` so that we can automatically populate these columns when creating a new entry.

Reply:
41c0ea3